mono/mini/aot-compiler.c

```diff
@@ -7163,6 +7169,9 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			opts->mode = MONO_AOT_MODE_FULL;
 		} else if (str_begins_with (arg, "hybrid")) {
 			opts->mode = MONO_AOT_MODE_HYBRID;			
+		} else if (str_begins_with (arg, "interp")) {
+			opts->mode = MONO_AOT_MODE_FULL;
```

Comment:
I wasn't the one adding `opts->mode = MONO_AOT_MODE_FULL` and `mono_aot_mode_is_full`, so I only suggest we keep the same mechanism for sake of consistency and readability.